sklearn/datasets/descr/twenty_newsgroups.rst

```diff
@@ -105,10 +114,10 @@ components by sample in a more than 30000-dimensional space
 (less than .5% non-zero features)::
 
   >>> vectors.nnz / float(vectors.shape[0])
-  159.01327...
```

Comment:
We generally don't expose so many digits. Try ``# doctest: +ELLIPSIS``

Reply:
did it. But I found that many more results expose all the digits. Should I make the same for all ?

Reply:
Yes, please.